README.md

```diff
@@ -1,13 +1,7 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
+The assignment was to create a form with react.js and useState. The form was to include radiobuttons, dropdowns and a summary. 
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I got the functionality to work quite quick, but it was a challange to understand why it worked and how it was connected. I will continue to try to understand it. 
 
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
-## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+I stuggeled to make the restart button to work. If I had more time I would like to clear the form when it restarts. I would like to develop the css more. 
```

Comment:
Might be a good idea to have the netlify link in the readme-file as well as the pull request reflections.